core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1808,6 +1810,34 @@ public ServerBuilder http1HeaderNaming(Http1HeaderNaming http1HeaderNaming) {
         return this;
     }
 
+    /**
+     * Sets the interval between log messages that report the
+     * exceptions not logged by any decorators or services, such as
+     * {@link LoggingService}.
+     * @param unloggedExceptionReportInterval the interval between the summary log messages, or
+     *                                        non-positive {@link Duration} to disable this feature
+     */
+    public ServerBuilder unloggedExceptionReportInterval(Duration unloggedExceptionReportInterval) {
+        requireNonNull(unloggedExceptionReportInterval, "unloggedExceptionReportInterval");
+        if (unloggedExceptionReportInterval.isNegative()) {
```

Comment:
We usually throw an `IllegalArgumentException` if the value is negative.
https://github.com/line/armeria/blob/554d5963bef33162b1cb077077336de3ef3aadfb/consul/src/main/java/com/linecorp/armeria/client/consul/ConsulEndpointGroupBuilder.java#L82
How about applying the same approach?

Reply:
Using the same approach you mentioned looks better 👍

Reply:
> I prefer "To be" way because it doesn't make much sense to have negative values for intervals.

I also prefer that. 😄

Reply:
SGTM